test/utils/replicaset.go

```diff
@@ -50,3 +51,18 @@ func UpdateReplicaSetWithRetries(c clientset.Interface, namespace, name string,
 	}
 	return rs, pollErr
 }
+
+// Verify .Status.Replicas is equal to .Spec.Replicas
+func WaitRSStable(t *testing.T, clientSet clientset.Interface, rs *extensions.ReplicaSet, pollInterval, pollTimeout time.Duration) error {
```

Comment:
@janetkuo I believe it is best to have `WaitRSStable` here. The util function is used by both replicaset and deployment, which is same with another util function named `UpdateReplicaSetWithRetries` that is located here. `UpdateReplicaSetWithRetries` is used by replicaset (for e2e) and deployment (for integration).